Soccar [Game-Server]/src/nl/soccar/gameserver/controller/rmi/GameServerForMainServer.java

```diff
@@ -12,12 +12,21 @@
 import nl.soccar.rmi.RmiConstants;
 
 /**
+ * A GameServerForMainServer is an RMI-stub object used by the Main Server that
+ * realizes the RMI-methods provided by the IGameServerForMainServer interface.
+ *
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Override this superclass' "equals" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS2160)